index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" 
+    integrity="sha384-UHRtZL1+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://bootswatch.com/4/yeti/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+
+
+    <title>Awesome Books</title>
+</head>
+
+<body>
+    <div class="container mt-4">
+        <h1>All Awesome Books</h1>
+        <table class="table table-striped mt-5">
+
+            <tbody id="book-list"></tbody>
+
+
```

Comment:
- [ ] Please remove the extra blank lines and indentations in the file to maintain your code and the website optimized.
_Kindly keep in mind that we're only adding them for good reasons such as separating two logical pieces of code or two large sections apart. 🙏 🥰_
_Please apply this to all similar situations through the file._

>You can find about this and more at this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

Reply:
nicely done 👍